docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
> I like the idea of users being able to define policy in their home dirs. We are working on atomic pull --user as we speak for running containers fully without privileges. But I worry about system policy being overridden by user policy.

As of https://github.com/projectatomic/skopeo/pull/157 users can do that explicitly via `skopeo --policy $any_path`.

As @wking says, this should mostly not give users any new privileges, though there are some a possibly worrisome cases to consider (`sudo skopeo copy` silently using the uprivileged `$HOME` might defeat attempts to audit changes to the system-wide policy file).

Again, please let’s not block adding this documentation on new feature requests. The documented semantics of `signature.DefaultPolicy` in #49 does gives us the leeway to implement this later, and for all users of the library to benefit transparently.